api/database/factories/ModelFactory.php

```diff
@@ -111,21 +111,21 @@
 $factory->define(App\Models\Article::class, function ($faker) {
     $publishedDatetime = $faker->optional(0.9)->dateTimeThisDecade();
 
-    $permalinks = [];
-
-    for ($i = rand(1, 10); $i >= 0; $i--) {
-        $permalinkObj = new \App\Models\ArticlePermalink();
-        $permalinkObj->permalink = str_random(255);
-        $permalinks[] = $permalinkObj;
-    }
-
-
     return [
         'article_id' => $faker->uuid,
         'title' => $faker->sentence,
         'content' => $content = implode("\n\n", $faker->paragraphs(3)),
-        'permalinks' => new \Spira\Repository\Collection\Collection($permalinks),
```

Comment:
I also removed this element as `permalinks` is not a direct property of the Articles model. Remember that the API documentation uses these model factories to generate dummy example request/responses, so the article's attributes should match the values.